docs/components/customization/organization-profile.mdx

```diff
@@ -1,416 +1,360 @@
 
-### Usage
+- `<OrganizationSwitcher />` tab: By default, the `<OrganizationSwitcher />` sets `organizationProfileMode='modal'`. If you are using the default settings, then you should select this tab.
+- `Dedicated page` tab: If you do not want the `<OrganizationProfile />` to open as a modal, then you should select this tab. For these examples, you need to set `organizationProfileMode='navigation'` and `organizationProfileUrl='/organization-profile'` on the `<OrganizationSwitcher />` component.
 
-<Tabs type="framework" items={["Next.js", "React", "Remix"]}>
-  <Tab>
-    `<OrganizationProfile.Page />` can be rendered **only on the client**. For Next.js applications using App Router, the `"use client";` directive must be added.
-
-    <CodeBlockTabs type="router" options={["App Router", "Pages Router"]}>
-      ```jsx filename="/app/organization-profile/[[...organization-profile]]/page.tsx"
-      "use client";
-
-      import { OrganizationProfile } from "@clerk/nextjs";
-      import { CustomProfilePage, CustomTerms } from "../components";
-      import { CustomIcon } from "../icons";
-
-      const OrganizationProfilePage = () => (
-        <OrganizationProfile path="/organization-profile" routing="path">
-          <OrganizationProfile.Page label="Custom Page" labelIcon={<CustomIcon />} url="custom-page">
-              <CustomProfilePage />
-          </OrganizationProfile.Page>
-          <OrganizationProfile.Page label="Terms" labelIcon={<CustomIcon />} url="terms">
-              <CustomTerms />
-          </OrganizationProfile.Page>
-        </OrganizationProfile>
-      );
-
-      export default OrganizationProfilePage;
-      ```
-
-      ```jsx filename="/pages/organization-profile/[[...index]].tsx"
-      import { OrganizationProfile } from "@clerk/nextjs";
-      import { CustomProfilePage, CustomTerms } from "../components";
-      import { CustomIcon } from "../icons";
-
-      const OrganizationProfilePage = () => (
-        <OrganizationProfile path="/organization-profile" routing="path">
-          <OrganizationProfile.Page label="Custom Page" labelIcon={<CustomIcon />} url="custom-page">
-              <CustomProfilePage />
-          </OrganizationProfile.Page>
-          <OrganizationProfile.Page label="Terms" labelIcon={<CustomIcon />} url="terms">
-              <CustomTerms />
-          </OrganizationProfile.Page>
-        </OrganizationProfile>
-      );
+For the sake of this guide, examples are written for Next.js App Router, but they are supported by any React meta framework, such as Remix or Gatsby.
 
-      export default OrganizationProfilePage;
-      ```
-    </CodeBlockTabs>
-  </Tab>
-
-  <Tab>
-    ```jsx filename="/organization-profile.tsx"
-    import { OrganizationProfile } from "@clerk/clerk-react";
-    import { CustomProfilePage, CustomTerms } from "../components";
-    import { CustomIcon } from "../icons";
+## Add a custom page to the `<OrganizationProfile />` component
```

Comment:
```suggestion
## Add a custom page to `<OrganizationProfile />`
```

While these headings are probably nice for search performance, they look super crowded in the ToC:

![image](https://github.com/clerk/clerk-docs/assets/5156915/fcf023d4-882f-413e-b49a-b3d6ecceda7b)

This and the other change suggestions just make the heading a little less crowded